firefox/flashblock.html

```diff
@@ -36,53 +36,72 @@
 
                     <table id="tests">
                         <tr>
                             <th>Test</th>
-                            <th>Expected</th>
+                            <th>Expected for<br>"Always Activate"</th>
+                            <th>Expected for<br>"Ask to Activate"</th>
+                            <th>Expected for<br>"Never Activate"</th>
                             <th>Actual</th>
                         </tr>
                         <tr>
                             <td>Top-Level</td>
+                            <td><img src="/img/flash-loaded.png"></td>
                             <td><img src="/img/flash-cta.png"></td>
-                            <td><object data="/swf/failure.swf"></object></td>
+                            <td><!-- Display nothing here. Blocked plugins are not displayed --></td>
+                            <td><object data="/swf/loaded.swf"></object></td>
                         </tr>
                         <tr>
                             <td>Blocked</td>
                             <td><!-- Display nothing here. Blocked plugins are not displayed --></td>
+                            <td><!-- Display nothing here. Blocked plugins are not displayed --></td>
+                            <td><!-- Display nothing here. Blocked plugins are not displayed --></td>
                             <td><iframe src="https://flashblock.itisatrap.org/firefox/flashblock-failure-frame.html" scrolling="no" frameBorder="0" seamless='seamless'></iframe></td>
                         </tr>
                         <tr>
                             <td>Block Exception</td>
+                            <td><img src="/img/flash-loaded.png"></td>
                             <td><img src="/img/flash-cta.png"></td>
-                            <td><iframe src="https://except.flashblock.itisatrap.org/firefox/flashblock-failure-frame.html" scrolling="no" frameBorder="0" seamless='seamless'></iframe></td>
+                            <td><!-- Display nothing here. Blocked plugins are not displayed --></td>
+                            <td><iframe src="https://except.flashblock.itisatrap.org/firefox/flashblock-loaded-frame.html" scrolling="no" frameBorder="0" seamless='seamless'></iframe></td>
                         </tr>
                         <tr>
                             <td>Allowed</td>
                             <td><img src="/img/flash-success.png"></td>
+                            <td><img src="/img/flash-success.png"></td>
+                            <td><!-- Display nothing here. Blocked plugins are not displayed --></td>
                             <td><iframe src="https://flashallow.itisatrap.org/firefox/flashblock-success-frame.html" scrolling="no" frameBorder="0" seamless='seamless'></iframe></td>
```

Comment:
I think this would should also be using `loaded-frame.html` otherwise the third expected column will show a "success" swf when the test fails.